core/org.openjdk.jmc.flightrecorder.writer/pom.xml

```diff
@@ -45,26 +45,58 @@
         <maven.compiler.target>8</maven.compiler.target>
         <maven.compiler.source>8</maven.compiler.source>
     </properties>
+
```

Comment:
I guess the copyright should be 2021 for both Oracle and Datadog in this new file.

Reply:
done